test/integration/common/util.go

```diff
@@ -200,3 +201,23 @@ func ValidateTraffic(f *framework.Framework, serverDeploymentBuilder *manifest.D
 	Expect(successRate).Should(BeNumerically(">=", succesRate))
 
 }
+
+func WaitToReconcileInitialState(f *framework.Framework, primaryInstance *ec2.Instance, defaultEniCount int, defaultIpsPerEni int, DefaultPrefixPerEni int) {
+	By("Verifying number of enis, ips and ip prefixes before updating the parameters")
+	Eventually(func(g Gomega) {
+
```

Comment:
nit: I would remove a lot of these empty newlines so that the related calls are connected